src/preprocessing.ml

```diff
@@ -477,10 +477,48 @@ let get_compat_ppx_exe sctx ~name ~kind =
     in
     ppx_exe sctx ~key ~dir_kind:Jbuild
 
-let get_ppx_driver sctx ~loc ~scope ~dir_kind pps =
+let args_of_cookies cookies =
+  List.map ~f:(fun (name, value) ->
+    ["--cookie"; sprintf "%s=%S" name value]
```

Comment:
@diml: do we want to support only "string cookies" of the shape : `name="some string"` or all kind in of cookies. 
If we support only string cookies we can use this printf, otherwise we have to use "%s=%s" instead and the user will have to write : (cookies (name "\"%{profile}\"")) instead of the plain (cookies (name "%{profile}")).

Reply:
Only string cookies. The syntax `(cookies (name "\"%{profile}\""))` feels too heavy and relies on `%{profile}` expanding to something that doesn't need escaping. If we wanted to support arbitrary expressions, we could support `(cookies (name (raw ...)))` in addition. Personally, I'm fine with supporting only string cookies.

Reply:
Ok, I'll try to present something reviewable in the next days if I find time !

Reply:
Looking forward to it! :)